srcpkgs/wsdd/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'wsdd'
+pkgname=wsdd
+version=0.6.4
+revision=1
+archs="*"
```

Comment:
You can simply omit the `archs` variable.

Reply:
Removed!